content/en/docs/File Format/implementationstatus.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Implementation status"
+linkTitle: "Implementation status"
+weight: 8
+---
+
+### Physical types
+
++-------------------------------------------+-------+--------+--------+-------+-------+
+| Data type                                 | C++   | Python | Java   | Go    | Rust  |
```

Comment:
I think this should be a bit more specific than simply listing programming languages, as there might be several C++ or Java implementations? Replace them with their actual names.

Also, I don't think Python needs to be included here, since PyArrow is a binding around Arrow C++ (unless this is talking about another implementation, such as [fastparquet](https://fastparquet.readthedocs.io/en/latest/))?

Reply:
Agreed on replacing with actual names.

I wasn’t sure it currently or historically pyarrow features match arrow c++. Likely it can be omitted from some tables. I could imagine eg int96/nanos/decimal (fixed array) capabilities or compression options differing from c++ and on high level API the difference was already confusing for me.

Reply:
Yeah, PyArrow might not expose everything. But that's more of a PyArrow problem and I'm not sure it's worth exposing here. cc @jorisvandenbossche for opinions.

Reply:
Actually the UUID is good counter example indeed :)

Reply:
We don't support writing UUID through Arrow data indeed. But it probably can be done directly through the Parquet C++ APIs (not PyArrow).